Library/Formula/kibana.rb

```diff
@@ -0,0 +1,63 @@
+class Kibana < Formula
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://download.elastic.co/kibana/kibana/kibana-4.0.2-darwin-x64.tar.gz"
+  version "4.0.2"
+  sha256 "3792e423dd97a216e12b0bb1ee97cf75712b2f850422d3746ff076ce9e1f2e01"
+  head "https://github.com/elastic/kibana.git"
+
+  def install
+    prefix.install %w[ node src ]
+
+    etc_path.mkpath
+
+    var_path.mkpath
+
+    conf_path = etc_path/"kibana.yml"
+
+    cp "config/kibana.yml", conf_path unless conf_path.exist?
+    (prefix/"config").install_symlink conf_path
+
+    (var_path/"plugins").mkpath
+    prefix.install_symlink var_path/"plugins"
+
+    bin.install "bin/kibana"
+  end
+
+  def caveats; <<-EOS.undent
+    Modify #{etc_path}/kibana.yml to set the elasticsearch_url to your Elasticsearch instance
```

Comment:
Let's default this to the Homebrew elasticsearch location and kill this comment.